labellab-flask/api/main.py

```diff
@@ -5,10 +5,9 @@
 from flask_jwt_extended import JWTManager
 
 from api.config import config
-from api.routes import users, models
-from api.extensions import db, migrate, jwt
-from api.models import User, Image, Label, LabelData, ProjectMembers, Projects, Team, RevokedToken, MLModel
-
+from api.routes import users, projects, models, images
```

Comment:
```project``` module is not found. Project routes are not yet defined, right?

Reply:
Project routes are defined in another pull request..:). If you want I can remove it from here